crates/bevy_core/src/time/fixed_timestep.rs

```diff
@@ -121,7 +121,10 @@ impl FixedTimestep {
     ) -> ShouldRun {
         let should_run = state.update(&time);
         if let Some(ref label) = state.label {
-            let res_state = fixed_timesteps.fixed_timesteps.get_mut(label).unwrap();
+            let res_state = fixed_timesteps
+                .fixed_timesteps
+                .get_mut(label)
+                .expect("Could not find fixed_timesteps");
```

Comment:
It can find "fixed_timesteps", just not with the specified label.

```suggestion
                .unwrap_or_else(|| {
                    panic!("Could not find a FixedTimestep labeled {label}")
                })
```